index.d.ts

```diff
@@ -26,6 +26,11 @@ declare module 'react-native-health' {
       callback: (err: string, results: HealthValue) => void,
     ): void
 
+    getBloodType(
```

Comment:
I don't how you tested this for your GIF, but here it is missing the   export for the method `RCT_EXPORT_METHOD`. Can you add this as well?

Reply:
Sorry, by accident didn't push that change

Reply:
Perfect!